docs/samples/autoscaling/README.md

```diff
@@ -0,0 +1,306 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Autoscale KFService with your inference workload](#autoscale-kfservice-with-your-inference-workload)
+  - [Setup](#setup)
+  - [Load your KFService with target concurrency](#load-your-kfservice-with-target-concurrency)
+    - [Create the KFService](#create-the-kfservice)
+    - [Load KFService with concurrent requests](#load-kfservice-with-concurrent-requests)
+    - [Check Dashboard](#check-dashboard)
+  - [Load your KFService with target QPS](#load-your-kfservice-with-target-qps)
+    - [Create the KFService](#create-the-kfservice-1)
+    - [Load KFService with target QPS](#load-kfservice-with-target-qps)
+    - [Check Dashboard](#check-dashboard-1)
+  - [Autoscaling on GPU!](#autoscaling-on-gpu)
+    - [Create the KFService with GPU resource](#create-the-kfservice-with-gpu-resource)
+    - [Load KFService with concurrent requests](#load-kfservice-with-concurrent-requests-1)
+  - [Autoscaling Customization](#autoscaling-customization)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+
+# Autoscale KFService with your inference workload
+## Setup
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/blob/master/docs/DEVELOPER_GUIDE.md#deploy-kfserving).
+2. Your cluster's Istio Ingress gateway must be network accessible.
+3. Your cluster's Istio Egresss gateway must [allow Google Cloud Storage](https://knative.dev/docs/serving/outbound-network-access/)
+4. [Metrics installation](https://knative.dev/docs/serving/installing-logging-metrics-traces) for viewing scaling graphs (optional).
```

Comment:
Metrics seems to be installed by default (at least for the 0.7 GCP docs I followed). Is it not default for some installs?